web/src/main/java/ua/com/vertex/logic/UserLogicImpl.java

```diff
@@ -103,6 +104,14 @@ public void registrationUserUpdate(User user) throws DataAccessException {
     }
 
     @Override
+    public Map<String, String> getTeachers() throws DataAccessException {
```

Comment:
Name of method supposed that List<User> would be returned... So can you please change the name?